crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -296,6 +304,55 @@ where
             status_code: if receipt.success { Some(U64::from(1)) } else { Some(U64::from(0)) },
         };
 
+        // get the previous transaction cumulative gas used
+        let prev_tx_cumulative_gas_used = match all_receipts.get((meta.index - 1) as usize) {
+            Some(prev_receipt) => {
+                if meta.index > 0 {
+                    prev_receipt.cumulative_gas_used
+                } else {
+                    0
+                }
+            }
+            None => 0,
+        };
+
+        res_receipt.state_root = if receipt.success && meta.block_number >= BYZANTIUM_FORK_BLKNUM {
+            Some(H256::from_low_u64_be(1))
+        } else {
+            Some(H256::from_low_u64_be(0))
+        };
```

Comment:
what's the logic behind this?

we should leave the state root blank for now

Reply:
Okay